libs/designer-ui/src/lib/dynamicallyaddedparameter/helper.ts

```diff
@@ -117,3 +118,50 @@ export function serialize(props: DynamicallyAddedParameterProps[]): ValueSegment
     },
   ];
 }
+
+export function createDynamicallyAddedParameterProperties(
+  itemType: DynamicallyAddedParameterTypeType,
+  description: string
+): IDynamicallyAddedParameterProperties {
+  let format, fileProperties;
+  let type = '';
```

Comment:
Would be nice to have swagger type enum for list of possible values

Reply:
on it